tools/example-showcase/src/main.rs

```diff
@@ -545,7 +546,7 @@ aliases = [\"/examples{}/{}/{}\"]
 technical_name = \"{}\"
 link = \"/examples{}/{}/{}\"
 image = \"../static/screenshots/{}/{}.png\"
-code_path = \"content/examples{}/{}\"
+code_path = \"content/examples{}/{}\"{}
```

Comment:
I would prefer something like
```suggestion
code_path = \"content/examples{}/{}\"
hader_code_paths = {:?}
```

Reply:
The reason I didn't do it this way was because not every example has shaders so I wasn't sure if we wanted to include the field for all of them. I asked on the discord and was told it was preferable this way.

Reply:
I think I like the separate field better, having seen this now.

Reply:
> I think I like the separate field better, having seen this now.

@alice-i-cecile I'm a little confused, it was always a seperate field but the text was added in the format (which François commented on a little bit down). The decision is between whether the `shader_code_paths` field should be included the `index.md` files of ALL examples, or just the shader ones. The way I had it originally was only for the shader examples but I'd be happy to change it.

Reply:
Follow Francois's advice!

Reply:
as they are auto generated it doesn't matter that it's more painful to write manually
and it means that on the parser / display side we'll need one less conditional to handle it as it will always be present